### PR TITLE
Feature/#006 imagenes calidad

### PR DESCRIPTION
**Pull request by IsmaelP19**
Added custom validator to check image quality when adding a url on the admin form.
Also modified fixtures to change initial values of images to take into accoutn the image quality requirement.

### CODE REVIEW

**Review by juradosalado — APPROVED** [review]
Images with bad quality are not able to be upload, and tests run with no failure. Good job!

File: requirements.txt [hunk 1]
```diff
@@ -4,3 +4,4 @@ django-countries
 stripe
 crispy-bootstrap5
 python-decouple
+pillow
```
Comment (ezegonmac) [inline requirements.txt:7]: I think pillow is not used in the project

**Review by ezegonmac — APPROVED** [review]
The validation works fine for creating and updating products. Delete pillow dependency if not used. Great job 👍🏻

File: requirements.txt (hunk 1 above)
Comment (IsmaelP19) [reply requirements.txt:7]: The pillow dependency is necessary to check the size of the images

**Pull request merged**